docs/source/accelerate_training.mdx

```diff
@@ -0,0 +1,91 @@
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Accelerating Training
+
+Gaudi offers several possibilities to make training faster.
+They are all compatible with each other and can be coupled with [distributed training](distributed).
+
+
+## Lazy Mode
+
+Two execution modes are proposed:
+- *Lazy mode*, where the Habana bridge internally accumulates operations in a graph. The execution of the operations in the accumulated graph is triggered in a lazy manner. This allows the bridge to construct a graph with multiple operations, which provides the graph compiler the opportunity to optimize the device execution for these operations.
+- *Eager mode*, where one operation at a time is executed.
+
+In lazy mode, the graph compiler generates optimized binary code that implements the given model topology on Gaudi. It performs operator fusion, data layout management, parallelization, pipelining and memory management, as well as graph-level optimizations.
+
+To execute your training in lazy mode, you must provide the following training arguments:
+```python
+args = GaudiTrainingArguments(
+    # same arguments as in Transformers,
+    use_habana=True,
+    use_lazy_mode=True,
+    gaudi_config_name=path_to_my_gaudi_config
+)
+```
+
+
+## Mixed-Precision Training
+
+Mixed-precision training enables to compute some operations using lighter data types to accelerate training.
+Habana Mixed Preicision (HMP) proposes to mix *fp32* and *bf16* operations.
+In order to do so, you must set `"use_habana_mixed_precision"` to `true` and `"hmp_opt_level"` to `"O1"` in the Gaudi configuration file.
+Then, you can specify which operators to compute in *bf16* with `"hmp_bf16_ops"` and which operators to compute in *fp32* with `"hmp_fp32_ops"`.
+If these operators are not specified, their default values are set to be the ones written in the [Gaudi configuration file of BERT](https://huggingface.co/Habana/bert-large-uncased-whole-word-masking/blob/main/gaudi_config.json), which is a good starting point for applying HMP:
+```
+"hmp_bf16_ops": [
+    "add",
+    "addmm",
+    "bmm",
+    "div",
+    "dropout",
+    "gelu",
+    "iadd",
+    "linear",
+    "layer_norm",
+    "matmul",
+    "mm",
+    "rsub",
+    "softmax",
+    "truediv"
+],
+"hmp_fp32_ops": [
+    "embedding",
+    "nll_loss",
+    "log_softmax"
+]
+```
+
+
+## Custom Operators
+
+### Fused ADAM
+
+Habana provides a [custom fused ADAM implementation](https://docs.habana.ai/en/v1.5.0/PyTorch/Model_Optimization_PyTorch/Custom_Ops_PyTorch.html#custom-optimizers).
```

Comment:
Is there a way to point to the latest docs an not `1.5.0` ?

Reply:
So in the end I changed all the links so that they point to the latest version as you suggested.

Reply:
Totally agree that aligned version would be nice but that's something you easily for get to update. And i think the goal is to always work with the latest synapseAI version

Reply:
Yes I agree with you. There should just be a few days between the release of a new version of SynapseAI and its added support in the library, so overall linking to the latest version of the doc is the most convenient way to deal with this.